clippy_lints/src/operators/arithmetic_side_effects.rs

```diff
@@ -222,6 +251,9 @@ impl<'tcx> LateLintPass<'tcx> for ArithmeticSideEffects {
             hir::ExprKind::AssignOp(op, lhs, rhs) | hir::ExprKind::Binary(op, lhs, rhs) => {
                 self.manage_bin_ops(cx, expr, op, lhs, rhs);
             },
+            hir::ExprKind::MethodCall(ps, instance, args, _) => {
+                self.manage_method_call(args, cx, instance, ps);
```

Comment:
IMO it would be better to use `receiver` in the method call.
https://doc.rust-lang.org/reference/expressions/method-call-expr.html#method-call-expressions
```suggestion
            hir::ExprKind::MethodCall(ps, receiver, args, _) => {
                self.manage_method_call(args, cx, receiver, ps);
```

Reply:
Done